packages/select/docs/FilterBarExample/DemographicValueSelect.tsx

```diff
@@ -74,8 +75,18 @@ export const DemographicValueSelect = ({
         <>
           <FilterMultiSelect.SearchInput />
           <FilterMultiSelect.ListBox>
-            {({ selectedItems, unselectedItems, disabledItems }) => (
+            {({
+              selectedItems,
+              unselectedItems,
+              disabledItems,
+              hasNoItems,
+            }) => (
               <>
+                {hasNoItems && (
+                  <FilterMultiSelect.NoResults>
+                    No results found.
+                  </FilterMultiSelect.NoResults>
+                )}
```

Comment:
Kinda annoying as a consumer to need to put this in 😞
Would it be possible to bake it into the `ListBox` or are there scenarios which would prevent it from working?

Reply:
It would be possible to put a generic message like this in there if there's no items, but...
* It wouldn't be possible to do the thing I did with the async one where it says "No results found for [search term].", because `ListBox` doesn't know anything about `SearchInput`
* It would go against the API pattern already established for this component, and in general it would feel weird for `ListBox` to show a message that more relates to `SearchInput` when they're not tightly coupled.
* Makes it less likely that consumers will implement a translated string for this. If there's something built in that looks like it works fine, they're gonna be more likely to just leave it.

Reply:
I guess in the case where the consumer can't get the input text to put into the message, I'd have expected Kaizen to translate it on their behalf.

Nonetheless, this is fine since the other points make enough sense for it the way it has been built :)